1.10/deploying-services/containerizers/docker-containerizer.md

```diff
@@ -1,25 +1,34 @@
 ---
-post_title: Docker Containerizer
+post_title: Docker Engine
 menu_order: 20
 ---
 
-Use the Docker containerizer if you need specific features of the Docker package. Refer to the [features matrix](/docs/1.10/deploying-services/containerizers/) to see if the Docker containerizer is the correct choice for your task.
+The [Docker Engine](https://www.docker.com/products/docker-engine) launches Docker containers from Docker images. Use the Docker Engine if you need features[ features](/docs/1.10/deploying-services/containerizers/#containerizer-runtime-features) of the Docker package. Otherwise, consider using the [Universal Container Runtime](/docs/1.10/deploying-services/containerizers/ucr).
```

Comment:
" of the Docker package" -> "of the Docker runtime.
I know it previous said package... but that is a strange word IMO.  It is the docker runtime features that folks may be interested in.